rest/route/agent.go

```diff
@@ -423,7 +423,8 @@ func (h *getExpansionsAndVarsHandler) Run(ctx context.Context) gimlet.Responder
 		})
 	}
 
-	appToken, err := h.settings.CreateGitHubAppAuth().CreateInstallationToken(ctx, pRef.Owner, pRef.Repo, nil)
+	const ghTokenLifetime = 50 * time.Minute
```

Comment:
is there a reason to use different life times for the agent and host create because 50vs59 seems pretty close and maybe we could create vars for shortLived = 15min, med = 30,  long lived=50min and we also wouldn't need a comment for every life time decision

Reply:
This one + host.create are only temporary placeholders until we get rid of the token expansion entirely, so I just tossed some arbitrary duration on them that would last a decently long time to avoid a regression. Once this is gone, there will only be 1 usage of the 50 minute lifetime (for git.get_project), so I don't think we need a constant for 1 use case.

I don't want to make top-level constants because for each usage, I actually had to assess it to determine that what it was using the token for, how long that operation could reasonably run for, and how often it's called (e.g. git clones need a token that lasts for a long time, but we don't need to give it a full hour). I don't think there would be much benefit in consolidating the numbers to match exactly since each one is a pretty specific decision.